tests/Issues/AutoImport/Fixture/with_html_start_normal_php_code.php.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+
+$this->headTitle('Transaction Detail');
+
+?>
+
+<div>
+    <?=new \Rector\Tests\Issues\AutoImport\Source\SomeClass()?>
+</div>
+-----
+<?php
+
+use Rector\Tests\Issues\AutoImport\Source\SomeClass;
+$this->headTitle('Transaction Detail');
+
+?>
+
+<div>
+    <?=new SomeClass()?>
+</div>
```

Comment:
Mix of PHP and HTML is quite a mess already Such transformations make code quite unreadable. 

The skip of any HTML node is on purpose, so we avoid mixed behavior of Rector when some rules work and some don't. Better skip them all together to make clear expectations we don't support HTML+PHP mix and people have to upgrade to MVC first :)